Source/WebCore/platform/unix/SharedMemoryUnix.cpp

```diff
@@ -76,11 +82,11 @@ static inline int accessModeMMap(SharedMemory::Protection protection)
     return PROT_READ | PROT_WRITE;
 }
 
-static UnixFileDescriptor createSharedMemory()
+static UnixFileDescriptor createSharedMemory([[maybe_unused]] size_t size)
```

Comment:
Isn't `UNUSED_PARAM(variable)` the standard way of doing this?

Reply:
New code has been introducing `[[maybe_unused]]` which [exists since C++17](https://en.cppreference.com/w/cpp/language/attributes/maybe_unused). There was a [discussion in webkit-dev](https://lists.webkit.org/pipermail/webkit-dev/2024-January/032609.html) and the consensus seemed to prefer the standard method over our custom macros.